docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
+each major release is a new major version (4.0.0)     continues "classic Ansible" versioning (2.11)
+follows semantic versioning rules                     does not use semantic versioning
+maintains only one version at a time                  maintains latest version plus two older versions
+includes language, runtime, and selected Collections  includes only the language and runtime
+developed and maintained in Collection repositories   developed and maintained in ansible/ansible repo
+====================================================  =================================================
+
+Most community users install the Ansible community package. The ``ansible-core`` option is primarily for developers.
 
 .. contents::
    :local:
 
 .. _release_cycle:
 
-Ansible release cycle
------------------------
+Release cycle overview
+======================
 
-Ansible is developed and released on a flexible release cycle.
-This cycle can be extended in order to allow for larger changes to be properly
-implemented and tested before a new release is made available. See :ref:`roadmaps` for upcoming release details.
+The two community releases are related - the release cycle follows this pattern:
 
-For Ansible version 2.10 or later, the major release is maintained for one release cycle. When the next release comes out (for example, 2.11), the older release (2.10 in this example) is no longer maintained.
+#. Release of a new ansible-core major version, for example, ansible-core 2.11
 
-If you are using a release of Ansible that is no longer maintained, we strongly
-encourage you to upgrade as soon as possible in order to benefit from the
-latest features and security fixes.
+   * New release of ansible-core and two prior versions are now maintained (ansible-base 2.10, Ansible 2.9)
+   * Work on new features for ansible-core continues in the ``devel`` branch
 
-Older, unmaintained versions of Ansible can contain unfixed security
-vulnerabilities (*CVE*).
+#. Collection freeze (no new Collections or new versions of existing Collections) on the Ansible community package
+#. Release candidate for Ansible community package, testing, additional release candidates as necessary
+#. Release of a new Ansible community package major version based on the new ansible-core, for example, Ansible 4.0.0 based on ansible-core 2.11
 
-You can refer to the :ref:`porting guides<porting_guides>` for tips on updating your Ansible
-playbooks to run on newer versions. For Ansible 2.10 and later releases, you can install the Ansible package with ``pip``. See :ref:`intro_installation_guide` for details.  For older releases, You can download the Ansible release from `<https://releases.ansible.com/ansible/>`_.
+   * Newest release of the Ansible community package is the only version now maintained
+   * Work on new features continues in Collections
+   * Individual Collections can make multiple minor and/or major releases
```

Comment:
```suggestion
   * Individual Collections can make multiple minor and/or major releases
```
I think it's worth mentioning somewhere (not exactly here) that an Ansible community package major release includes latest major collection releases and keep them until next Ansible community package major release, so only minor and bugfix collection releases will be accepted between Ansible community package major releases.
@felixfontein , am i correct?

Reply:
I can see a bit below something about it but it doesn't seem to be explicitly said

Reply:
I changed thttps://github.com/ansible/ansible/pull/73781#discussion_r587344064 a little bit to clarify this

Reply:
I changed thttps://github.com/ansible/ansible/pull/73781#discussion_r587344064 a little bit to clarify this